jboss-fuse-sap-tool-suite/uitests/tests/org.jboss.tools.fuse.sap.ui.bot.tests/pom.xml

```diff
@@ -16,41 +16,51 @@
 		<systemProperties>${integrationTestSystemProperties}</systemProperties>
 		<customization.file>resources/settings/plugin_customization.ini</customization.file>
 		<staging.repos>false</staging.repos>
-		<skipUITests>false</skipUITests>
+		<skipUITests>true</skipUITests>
 		<testUIClass>SmokeTests</testUIClass>
 		<test.installPath.OSX/>
 	</properties>
-	<build>
-		<plugins>
-			<!-- Fuse SAP RedDeer Tests -->
-			<plugin>
-				<groupId>org.eclipse.tycho</groupId>
-				<artifactId>tycho-surefire-plugin</artifactId>
-				<configuration>
-					<appArgLine>-eclipse.password resources/security/password -pluginCustomization ${customization.file} -clean</appArgLine>
-					<testSuite>org.jboss.tools.fuse.sap.qe.reddeer.tests</testSuite>
-					<testClass>**/${testUIClass}</testClass>
-					<useUIThread>false</useUIThread>
-					<skip>${skipUITests}</skip>
-					<dependencies combine.children="append">
-						<!-- SAP Library Feature -->
-						<dependency>
-							<type>p2-installable-unit</type>
-							<artifactId>com.sap.conn.feature.group</artifactId>
-							<version>0.0.0</version>
-						</dependency>
-						<!-- This entry should enable creating of default JDK on Mac -->
-						<dependency>
-							<type>p2-installable-unit</type>
-							<artifactId>org.eclipse.jdt.feature.group</artifactId>
-							<version>0.0.0</version>
-						</dependency>
-					</dependencies>
-				</configuration>
-			</plugin>
-		</plugins>
-	</build>
 	<profiles>
+		<profile>
+			<id>surefire-ui-test-activation</id>
+			<activation>
+				<property>
+					<name>skipUITests</name>
+					<value>false</value>
```

Comment:
why is here explicitly defined "false"? can we use here $skipUITests variable defined in properties section?

Reply:
it is to activate or not by default the UI bot test.
They need to be deactivated by default to fit in the Jboss Tools templates.

i reused the $skipUITests defined in properties section to enable/disable the UI bot tests. Just now the yare disable dby default . to launch them we need to specify -DskipUITests=false

Reply:
yep, make sense, Thanks for clarification 👍